src/Sylius/Component/Payment/Calculator/PercentFeeCalculator.php

```diff
@@ -23,6 +23,9 @@ class PercentFeeCalculator implements FeeCalculatorInterface
      */
     public function calculate(PaymentSubjectInterface $payment, array $configuration)
     {
+        if (empty($configuration['percent'])) {
+            $configuration['percent'] = 0;
```

Comment:
The proper check will be ``` if (!isset($configuration['percent'])) ``` then we should throw exception instead of setting value to null (percent key should always be present).